src/apps/Settings/DaoSettings.js

```diff
@@ -0,0 +1,192 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Button, Field, TextInput, Text } from '@aragon/ui'
+import EtherscanLink from '../../components/Etherscan/EtherscanLink'
+import provideNetwork from '../../context/provideNetwork'
+import { sanitizeNetworkType } from '../../network-config'
+import { compose } from '../../utils'
+import { getWeb3 } from '../../web3-utils'
+import { web3Providers, network, appIds } from '../../environment'
+import airdrop, { testTokensEnabled } from '../../testnet/airdrop'
+import Option from './Option'
+import Note from './Note'
+
+const LinkButton = ({ children }) => (
+  <Button.Anchor mode="outline" compact>
+    {children}
+  </Button.Anchor>
+)
+
+// const LinkButton = styled(Button.Anchor).attrs({
+// compact: true,
+// mode: 'outline',
+// })`
+// background: ${theme.contentBackground};
```

Comment:
I think I needed this override to make the background white—is it not needed anymore?